File: testutils-logrecorder/src/main/java/info/novatec/testit/testutils/logrecorder/api/LogLevel.java
package info.novatec.testit.testutils.logrecorder.api;

/**
 * All the supported log levels. This should cover most of the logging frameworks used today. Any level not matching one of
 * these should be mapped to {@link #UNKNOWN}.
 */
public enum LogLevel {
    UNKNOWN,
    TRACE,
    DEBUG,
    INFO,
    WARN,
    ERROR;
}

File: logrecorder/logrecorder-api/src/main/kotlin/info/novatec/testit/logrecorder/api/LogLevel.kt
package info.novatec.testit.logrecorder.api

/**
 * All the supported log levels.
 *
 * This should cover most of the logging frameworks used today.
 * Any level not matching one of these should be mapped to [UNKNOWN].
 *
 * @since 1.0
 */
enum class LogLevel {
    TRACE, DEBUG, INFO, WARN, ERROR, UNKNOWN
}
